urlconnection/src/test/java/zipkin/reporter/urlconnection/URLConnectionSenderTest.java

```diff
@@ -184,7 +233,19 @@ void thenCallbackCatchesTheThrowable() {
   /** Blocks until the callback completes to allow read-your-writes consistency during tests. */
   void send(List<Span> spans) {
     AwaitableCallback callback = new AwaitableCallback();
-    sender.sendSpans(spans.stream().map(Encoder.THRIFT::encode).collect(toList()), callback);
```

Comment:
I think it would be simpler to make Encoder a field in the test, and just change to map (encoder::encoding) right?